docs/user_guides/fs/feature_group/advanced_data_validation.md

```diff
@@ -0,0 +1,276 @@
+
+If you need to set or edit the existing requirements, you can go on the Feature Group edit page. The Expectation suite section allows you to edit individual expectations and set success parameters that match ever changing business requirements.
+
+### Production
+
+Models in production require high-quality data to make accurate predictions for your customers. Hopsworks can use your Expectation Suite as a gatekeeper to make it simple to prevent low-quality data to make its way into production. Below are some simple tips and snippets to make the most of your data validation when your project is ready to enter its production phase.
+
+#### Be Strict in Production
+
+Whether you use an existing or create a new (recommended) Feature Group for production, we recommend you set the validation ingestion policy of your Expectation Suite to `"STRICT"`.
+
+```python3
+fg_prod.save_expectation_suite(
+    my_suite,
+    validation_ingestion_policy="STRICT")
+```
+
+In this setup, Hopsworks will abort inserting a DataFrame that does not successfully fullfill all expectations in the attached Expectation Suite. This ensures data quality standards are upheld for every insertion and provide downstream users with strong guarantees.
+
+#### Avoid Data Loss on Backfill jobs
+
+Aborting insertions of DataFrames which do not satisfy the data quality standards can lead to data loss in your backfill job. To avoid such loss we recommend creating a duplicate Feature Group with the same Expectation Suite in `"ALWAYS"` mode which will hold the rejected data.
+
+```python3
+job, report = fg_prod.insert(df)
+
+if report["success"] is False:
+    job, report = fg_rejected.insert(df)
+```
+
+#### Take Advantage of the Validation History
+
+You can easily retrieve the validation history of a specific expectation to export it to your favourite visualisation tool. You can filter on time and on whether insertion was successful or not
+
+```python3
+validation_history = fg.get_validation_history(
+    expectation_id=my_id,
+    filters=["REJECTED", "UNKNOWN"],
+    ge_type=False
+)
+
+timeseries = pd.DataFrame(
+    {
+        "observed_value": [res.result["observed_value"] for res in validation_histoy]],
+        "validation_time": [res.validation_time for res in validation_history]
+    }
+)
+
+# export to your preferred Dashboard
+```
+
+#### Setup Alerts
+
+While checking your feature engineering pipeline executed properly in the morning can be good enough in the development phase, it won't make the cut for demanding production use-cases. In Hopsworks, you can setup alerts if validation fails.
+
+First you will need to configure your preferred communication endpoint: slack, email or pagerduty. Check out [this page](../../../admin/alert.md) for more information on how to set it up. You can then set an alert on data ingestion success in the Feature Group holding rejected data for example.
```

Comment:
Do we have any documentation or example for `You can then set an alert on data ingestion success in the Feature Group holding rejected data for example.` Because at this point I have no idea if the alert should be set in the admin page or if it is in the feature group page or somewhere else.